### PR TITLE
Reduce to a single (hopefully fast) build-containers

### PR DESCRIPTION
**Pull request by edwarnicke**
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -182,17 +180,7 @@ jobs:
...
-
-# build ready - empty job
-  build-container-success:
-    docker:
-      - image: circleci/golang:1.13-stretch
-    steps:
-      - run:
-          command: |
-            echo "SUCCESS"
+            make -j 8 docker-push
```
Comment (alex-yust) [inline .circleci/config.yml]: We can use just '-j' option (no concrete number) complemented by load restriction option '-l'.
Also, log aggregation option '--output-sync=line' might be a useful addition
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_toc.html#TOC47

File: .mk/k8s.mk [hunk 2]
```diff
@@ -56,7 +56,7 @@ SPIRE_ENABLED=false
 endif
 
 kubectl = kubectl -n ${NSM_NAMESPACE}
-images_tar = $(subst .tar,,$(filter %.tar, $(shell ls ./scripts/vagrant/images)))
+images_tar = $(subst .tar,,$(filter %.tar, $(shell mkdir -p ./scripts/vagrant/images;ls ./scripts/vagrant/images)))
```
Comment (haiodo) [inline .mk/k8s.mk]: Actually I think we need to move from using of ./scripts/**vagrant** folder as a place for docker images

Comment (alex-yust) [reply .mk/k8s.mk]: > we need to move from using of ./scripts/vagrant folder as a place for docker images

+1
We might also want a variable in build.mk for this, like IMAGE_DIR

File: .circleci/config.yml (hunk 1 above)
Comment (edwarnicke) [reply .circleci/config.yml]: The physical machine underlying has 8 cores.  I briefly experimented with -j 16, but it didn't help much.  If you have more advice on how to tune, I'd love to take it :)

As to --output-sync=line ... this is a very good idea... why line and not target though?

File: .mk/k8s.mk (hunk 2 above)
Comment (edwarnicke) [reply .mk/k8s.mk]: Yeah... we need a better way here... I took the path of the minimum fix that actually fixed the breakage in order to get to consolodated builds.  I'm very open to a better approach to that.

What I *do* like about the images_tar line is that it attempts to dynamically figure out the images_tars ... that's a very good trend in our Makefiles :)

**Pull request merged**